widgets/WebComponents/AvatarIcon/AvatarIcon.bbj

```diff
@@ -0,0 +1,46 @@
+use ::WebKit/widgets/WebComponents/WebComponents.bbj::WebComponents
+use ::WebKit/util/ClientUtil.bbj::ClientUtil
+
+class public AvatarIcon extends WebComponents
```

Comment:
This class is missing some setters to update the avatar values - See BBjWidget pattern